### PR TITLE
Fixes #54 Added the Instructions to follow during GSSOC

### LINKED ISSUE #54: Add 'instructions for gssoc' section in README.md
Add 'instructions for gssoc' file to the repo.
The intention of this issue is to solve the redundant queries asked about gssoc by the participants. For example, the working of 'taken' label. the guidelines to follow for contribution under gssoc.

### PR DESCRIPTION
**Pull request by HarshCasper**
Issue: https://github.com/coderwassananmol/BUB2/issues/54

File Name: Instructions_for_GSSOC.md

Intent: 
 
To add the instructions for GirlScript Summer of Code. 

Kindly review the Pull Request and let me know of any changes needed @Parikshit-Hooda @coderwassananmol

### CODE REVIEW

**Comment by HarshCasper** [comment]
I have updated the files. Kindly look into it @rkpattnaik780

**Comment by Parikshit-Hooda** [comment]
@HarshCasper please tend to the review.

File: Instructions_for_GSSOC.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Instructions for Girlscript Summer of Code 2020
+- Introduce yourself on the Slack Channel.
+- Get in touch with Mentors(M) on Slack, if you are new to Git and Github and wish to contribute.
+- Review the Issues which are tagged with GSSOC.
```
Comment (rkpattnaik780) [inline Instructions_for_GSSOC.md]: Labelled `gssoc20`

File: Instructions_for_GSSOC.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Instructions for Girlscript Summer of Code 2020
+- Introduce yourself on the Slack Channel.
+- Get in touch with Mentors(M) on Slack, if you are new to Git and Github and wish to contribute.
+- Review the Issues which are tagged with GSSOC. 
+- Before writing code do follow the CONTRIBUTING.md to follow the certain guidelines that must be adhered while contributing to the Project.
```
Comment (rkpattnaik780) [inline Instructions_for_GSSOC.md]: Before contributing to the codebase, go through `CONTRIBUTING.md` to know the guidelines that must be followed/adhered.

File: Instructions_for_GSSOC.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+- Also follow the README.md File to know about the project in depth.
```
Comment (rkpattnaik780) [inline Instructions_for_GSSOC.md]: README.md should be mentioned before CONTRIBUTING.md and it has instructions to locally run the project.

File: Instructions_for_GSSOC.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+- Make sure that you are choosing issues which have not be assigned the **taken** label. The **taken** label signifies that the issue has already been assigned to someone else.
+- If you wish to log a Bug Report or a Feature Request, follow the template followed for the same to help Mentors better understand your
+issue.
+- If you wish to work on an existing issue, check if the issue has already been assigned to someone else.
```
Comment (rkpattnaik780) [inline Instructions_for_GSSOC.md]: Line 7 and 10 seem redundant.

File: Instructions_for_GSSOC.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+- Introduce yourself on the Slack Channel.
+- Get in touch with Mentors(M) on Slack, if you are new to Git and Github and wish to contribute.
+- Review the Issues which are tagged with GSSOC. 
+- Before writing code do follow the CONTRIBUTING.md to follow the certain guidelines that must be adhered while contributing to the Project.
+- Also follow the README.md File to know about the project in depth.
+- Make sure that you are choosing issues which have not be assigned the **taken** label. The **taken** label signifies that the issue has already been assigned to someone else.
+- If you wish to log a Bug Report or a Feature Request, follow the template followed for the same to help Mentors better understand your
+issue.
+- If you wish to work on an existing issue, check if the issue has already been assigned to someone else. 
+- If you find no-one has been assigned an issue, you can comment your interest to work on the issue. You can start working on the issue after you are assigned.
```
Comment (rkpattnaik780) [inline Instructions_for_GSSOC.md]: I think the lines after this have been covered in `CONTRIBUTING.md`

**Comment by rkpattnaik780** [comment]
Closing this PR as it is taking a longer time than expected.

**Comment by HarshCasper** [comment]
@rkpattnaik780 Can you please open this PR since I had my Mid-Semester Exams during this period, and I could not devote much time to it. I will be much obliged if you do so.

**Pull request closed without merging**